src/components/control-bar/control-bar-items/AboutDialog.svelte

```diff
@@ -0,0 +1,52 @@
+<!--
+  (c) 2023, Center for Computational Thinking and Design at Aarhus University and contributors
+
+  SPDX-License-Identifier: MIT
+ -->
+
+<script lang="ts">
+  import StandardButton from '../../StandardButton.svelte';
+  import StandardDialog from '../../dialogs/StandardDialog.svelte';
+  import microbitLogo from '../../../imgs/microbit-logo-black.svg';
+  import microbitHeartImage from '../../../imgs/microbit-heart.png';
+  import aarhusLogo from '../../../imgs/aulogo_uk_var2_blue.png';
+  import GitHubIcon from 'virtual:icons/ri/github-fill';
+  import { t } from '../../../i18n';
+
+  export let onClose: () => void;
+  export let isOpen: boolean;
+</script>
+
+<StandardDialog {isOpen} {onClose}>
+  <div class="flex flex-col gap-5 w-150 text-black">
+    <div class="flex flex-col gap-8">
+      <div class="grid grid-cols-2 gap-20 items-center">
+        <img src={microbitLogo} alt="micro:bit" />
```

Comment:
Use of logos and form of words here all subject to discussion. But we'll merge as a starting point.